src/redempt/redlib/region/CuboidRegion.java

```diff
@@ -317,6 +319,32 @@ public CuboidRegion getFace(BlockFace face) {
 		region.expand(face.getOppositeFace(), -region.measure(face) + 1);
 		return region;
 	}
+
+	/**
+	 * Gets a random block from within the cuboid region
+	 * @return Random bukkit block from within the region
+	 */
+	public Block getRandomBlock() {
+		int[] limits = getBlockDimensions();
+		ThreadLocalRandom threadLocalRandom = getLocalRandom();
```

Comment:
Instead of copy-pasting the logic, why don't you just pass the ThreadLocalRandom to the variant of getRandomBlock that takes a Random?